web/cobrands/hampshire/assets.js

```diff
@@ -0,0 +1,63 @@
+(function(){
+
+if (!fixmystreet.maps) {
+    return;
+}
+
+var wfs_host = fixmystreet.staging ? 'tilma.staging.mysociety.org' : 'tilma.mysociety.org';
+var tilma_url = "https://" + wfs_host + "/mapserver/hampshire";
+
+var defaults = {
+    http_options: {
+        url: tilma_url,
+        params: {
+            SERVICE: "WFS",
+            VERSION: "1.1.0",
+            REQUEST: "GetFeature",
+            SRSNAME: "urn:ogc:def:crs:EPSG::27700"
+        }
+    },
+    geometryName: 'msGeometry',
+    srsName: "EPSG:27700",
+    body: "Hampshire County Council",
+    strategy_class: OpenLayers.Strategy.FixMyStreet
+};
+
+fixmystreet.assets.add(defaults, {
+    http_options: {
+        params: {
+            TYPENAME: "Road_Sections"
+        }
+    },
+    usrn: {
```

Comment:
You removed styling here, it needs some invisible styling as otherwise you get visible black lines, as you can see?